minecraft_servers/NinjaMC/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "server_name": "NinjaMC",
+  "nice_name": "NinjaMC.net",
+  "direct_ip": "NinjaMC.net",
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "discord": "https://social.dc.ninjamc.net"
```

Comment:
Discord link has to be https://discord.gg/[YOUR INVITE]
Custom links are for partner servers only